### PR TITLE
Telegraf 1.19.2

### LINKED ISSUE #2926: Telegraf 1.19.2 patch release
https://github.com/influxdata/telegraf/blob/master/CHANGELOG.md

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #2926

### CODE REVIEW

**Comment by sanderson** [comment]
Be sure update the latest patch version for Telegraf 1.19 in `data/products.yml`.

File: content/telegraf/v1.19/about_the_project/release-notes-changelog.md [hunk 1]
```diff
@@ -10,6 +10,43 @@ menu:
...
 ---
 
+## v.1.19.2 [2021-07-28]
+
+- Update Go to 1.16.6.
+- Linter fixes.
+- Update `dynatrace-metric-utils-go` module to v0.2.0.
+- Detect changes to configuration and reload Telegraf.
+
+## Input Plugin Updates
```
Comment (kelseiv) [inline content/telegraf/v1.19/about_the_project/release-notes-changelog.md:20]:
```suggestion
## Input plugin updates
```

File: content/telegraf/v1.19/about_the_project/release-notes-changelog.md [hunk 2]
```diff
@@ -10,6 +10,43 @@ menu:
...
+- NSQ Consumer (`couchbase`): Fix a connection error when try to connect to an empty list of servers.
+- Prometheus (`prometheus`): Fix Prometheus cAdvisor authentication.
+- SQL (`sql`): Fix issue when handling a boolean column.
+- SQL Server (`sqlserver`):
+  - Add TempDB troubleshooting stats and missing v2 query metrics.
+  - Update to provide more detailed error messaging.
+- StatsD (`statsd`): Fix a regression that didn't allow integer percentiles.
+- x509 Certificate (`x509_cert`): Fix an issue where plugin would hang indefinitely to a UDP connection.
+
+## Output Plugin Updates
```
Comment (kelseiv) [inline content/telegraf/v1.19/about_the_project/release-notes-changelog.md:34]:
```suggestion
## Output plugin updates
```

File: content/telegraf/v1.19/about_the_project/release-notes-changelog.md [hunk 3]
```diff
@@ -10,6 +10,43 @@ menu:
...
+- Dynatrace Output (`dynatrace`):
+  - Updated plugin to allow optional default dimensions
+  - Fixed a panic caused by uninitialized `loggedMetrics` map
+- InfluxDB (`influxdb`): Fix issue where metrics were reporting as written but not actually written.
+
+## Processor Plugin Updates
```
Comment (kelseiv) [inline content/telegraf/v1.19/about_the_project/release-notes-changelog.md:40]:
```suggestion
## Processor plugin updates
```

File: content/telegraf/v1.19/about_the_project/release-notes-changelog.md [hunk 4]
```diff
@@ -10,6 +10,43 @@ menu:
...
+- IfName (`ifname`): Fix issue with SNMP empty metric name.
+
+## Parser Plugin Updates
```
Comment (kelseiv) [inline content/telegraf/v1.19/about_the_project/release-notes-changelog.md:43]:
```suggestion
## Parser plugin updates
```

File: content/telegraf/v1.19/about_the_project/release-notes-changelog.md [hunk 5]
```diff
@@ -10,6 +10,43 @@ menu:
...
+## v.1.19.2 [2021-07-28]
+
+- Update Go to 1.16.6.
+- Linter fixes.
+- Update `dynatrace-metric-utils-go` module to v0.2.0.
+- Detect changes to configuration and reload Telegraf.
+
+## Input Plugin Updates
+- CGroup (`couchbase`): Allow for multiple keys when parsing cgroups.
+- Kubernetes (`kubernetes`): Update plugin to attach pod labels to the `kubernetes_pod_volume` & `kubernetes_pod_network` metrics.
```
Comment (kelseiv) [inline content/telegraf/v1.19/about_the_project/release-notes-changelog.md:22]:
```suggestion
- Kubernetes (`kubernetes`): Update plugin to attach pod labels to the `kubernetes_pod_volume` and `kubernetes_pod_network` metrics.
```

File: content/telegraf/v1.19/about_the_project/release-notes-changelog.md [hunk 6]
```diff
@@ -10,6 +10,43 @@ menu:
...
+- Kubernetes Inventory (`kube_inventory`): Fix a segmentation fault when selector labels were not present on a persistent volume claim.
+- MongoDB (`mongodb`): Switch to official `mongo-go-driver` module to fix an SSL authentication failure.
+- NSQ Consumer (`couchbase`): Fix a connection error when try to connect to an empty list of servers.
```
Comment (kelseiv) [inline content/telegraf/v1.19/about_the_project/release-notes-changelog.md:25]:
```suggestion
- NSQ Consumer (`couchbase`): Fix a connection error when attempting to connect to an empty list of servers.
```

File: content/telegraf/v1.19/about_the_project/release-notes-changelog.md [hunk 7]
```diff
@@ -10,6 +10,43 @@ menu:
...
+- SQL (`sql`): Fix issue when handling a boolean column.
+- SQL Server (`sqlserver`):
+  - Add TempDB troubleshooting stats and missing v2 query metrics.
+  - Update to provide more detailed error messaging.
+- StatsD (`statsd`): Fix a regression that didn't allow integer percentiles.
+- x509 Certificate (`x509_cert`): Fix an issue where plugin would hang indefinitely to a UDP connection.
+
+## Output Plugin Updates
+- Dynatrace Output (`dynatrace`):
+  - Updated plugin to allow optional default dimensions
```
Comment (kelseiv) [inline content/telegraf/v1.19/about_the_project/release-notes-changelog.md:36]:
```suggestion
  - Update plugin to allow optional default dimensions.
```

File: content/telegraf/v1.19/about_the_project/release-notes-changelog.md [hunk 8]
```diff
@@ -10,6 +10,43 @@ menu:
...
+  - Fixed a panic caused by uninitialized `loggedMetrics` map
```
Comment (kelseiv) [inline content/telegraf/v1.19/about_the_project/release-notes-changelog.md:37]:
```suggestion
  - Fix a panic caused by uninitialized `loggedMetrics` map.
```

**Review by kelseiv — APPROVED** [review]
Minor edits

**Review by sjwang90 — APPROVED** [review]

**Pull request merged**